src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -33,14 +33,22 @@ class ConfirmationPage extends React.Component {
           <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
         </div>
         <div>
-          <p>We are processing your application. The Department of Veterans Affairs will contact you when we finish our review.</p>
+          <p>
+            We are processing your application.<br/>
+            The Department of Veterans Affairs will contact you when we finish our review.<br/>
+            <i>Please print this page for your records.</i>
+          </p>
```

Comment:
I thought this might look better.
<img width="786" alt="screen shot 2017-09-25 at 4 44 49 pm" src="https://user-images.githubusercontent.com/16051172/30836222-94b929c8-a211-11e7-85ab-8975bee60223.png">
Opposed to:
<img width="777" alt="screen shot 2017-09-25 at 4 45 28 pm" src="https://user-images.githubusercontent.com/16051172/30836228-9c0c8526-a211-11e7-9d90-1b07bc6ea5db.png">
Please let me know if this is not necessary or if there are other ideas for how this should look.

Reply:
@cehsu Are these different screenshots? They look the same to me.

Reply:
That looks fine to me. We're also going to be changing the design of this page soon anyway.

Reply:
Thanks @jbalboni, the screenshot for the currently proposed version is below:
<img width="786" alt="screen shot 2017-09-25 at 4 44 49 pm" src="https://user-images.githubusercontent.com/16051172/30875538-b5d5f284-a2a8-11e7-991c-da67b080e927.png">